std/hints.go

```diff
@@ -7,14 +7,13 @@ import (
 	"github.com/consensys/gnark/std/math/bits"
 )
 
-// GetHints return std hints that are always injected in gnark solvers
-func GetHints() []hint.Function {
-	return []hint.Function{
-		sw_bls24315.DecomposeScalar,
-		sw_bls12377.DecomposeScalar,
-		bits.NTrits,
-		bits.NNAF,
-		bits.IthBit,
-		bits.NBits,
-	}
+// RegisterHints register std hints that are always injected in gnark solvers
+func RegisterHints() {
```

Comment:
It seems that this function is not referred anywhere. Should there be an example, maybe? If i recall, then this function can not be called in compiler/prover due to import cycles.